interfaces/Portal/src/app/components/update-property-item/update-property-item.component.html

```diff
@@ -25,14 +25,17 @@
   method="POST"
   #updatePropertyForm="ngForm"
 >
-  <div class="input-item update-property-item">
-    <label
-      data-testid="update-property-item-label"
-      class="input-item--label"
-    >
+  <div
+    data-testid="update-property-item-input-form"
+    class="input-item update-property-item"
+  >
+    <label class="input-item--label">
       <strong>{{ label }}</strong>
     </label>
-    <div class="input-item--contents">
+    <div
+      data-testid="update-property-item-label"
```

Comment:
Was this move actually intentional?

As I can imagine this will break some tests that want to verify/filter for a specific label-text being shown...

Reply:
This testid is currently not used so I do not know at this moment how I wanted to use it

Reply:
If it isn't used (anymore), I think it is best to remove it.

I think the only user was the `this.personAffectedCustomAttribute` from the `RegistrationDetails`-`Page`...

And that one seems now replaced by the new "update-property-item-input-form"...